### PR TITLE
Iso arm64

### PR DESCRIPTION
**Pull request by Itxaka**

### CODE REVIEW

**Comment by Itxaka** [comment]
Ready for review @mudler

File: make/Makefile.iso [hunk 1]
```diff
@@ -95,6 +95,7 @@ else
 	$(YQ) e -i '.luet.repositories[0].enable=true' $(MANIFEST).remote
 	$(YQ) e -i '.luet.repositories[0].priority=90' $(MANIFEST).remote
 	$(YQ) e -i '.luet.repositories[0].type="docker"' $(MANIFEST).remote
+	$(YQ) e -i '.luet.repositories[0].type="docker"' $(MANIFEST).remote
```
Comment (mudler) [inline make/Makefile.iso:98]: mm this looks like a leftover?

Comment (Itxaka) [reply make/Makefile.iso:98]: :thinking: weird

**Review by mudler — APPROVED** [review]
Looks good! That's great!

File: packages/livecd/grub2/build.yaml [hunk 2]
```diff
@@ -1,28 +1,35 @@
-image: opensuse/leap:15.3
+image: {{ .Values.image }}
```
Comment (mudler) [inline packages/livecd/grub2/build.yaml:1]: If we are switching image here, the `zypper` call below should be adjusted to take into account blue and orange

Comment (Itxaka) [reply packages/livecd/grub2/build.yaml:1]: indeed, Im wondering if it makes sense to have this package in other flavors....

**Review by mudler — CHANGES_REQUESTED** [review]
Just some nits, otherwise looks good

File: packages/livecd/grub2/build.yaml (hunk 2 above)
Comment (mudler) [reply packages/livecd/grub2/build.yaml:1]: Previously we were using files from opensuse for the other flavors too, we did that to let the user just specify one repo for flavor.. maybe we can do the same on aarch64 too?

**Comment by Itxaka** [comment]
arm iso built!

**Review by mudler — APPROVED** [review]

**Pull request merged**